src/System.Binary.Base64/System/Binary/Base64Encoder.cs

```diff
@@ -103,6 +105,8 @@ private static int EncodePadByTwo(ref byte srcBytes)
         /// <param name="source"></param>
         /// <param name="destination"></param>
         /// <returns>Number of bytes written to the destination.</returns>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        [Obsolete("Use Base64.BytesToUtf8InPlace")]
```

Comment:
Question: Why are these methods marked as Obsolete/hidden rather than them being internal or even private?

Reply:
I don't want to break Signal-R. Once they move to the new APIs, I will remove these members.